### DISCUSSION TITLE
Create alerts within the dagster UI

### DISCUSSION THREAD

**Discussion by gnilrets** [dagster-io/dagster#18122] [Ideas]
I had an error with my [slack failure sensor](https://github.com/dagster-io/dagster/issues/18121) that went on for several weeks before I noticed.  I thought it would be nice if there was a way to mark certain dagster objects (jobs, assets, or sensors) that would raise an alert notification within the dagster UI.  Any notification method we might have that talks to external services could fail.  As a fallback, if I got alerted about failures when I logged in to the dagster UI, I could catch them much faster.  I wouldn't necessarily want to be alerted for every failure, just ones that I mark as critical.

**Comment by tacastillo** [comment]
Hey Sterling! This is a great idea and I just reposted this as an issue so we can track it! #18130
